agent/ecr/model/generate.go

```diff
@@ -13,4 +13,4 @@
 
 package model
 
-//go:generate go run ../../gogenerate/awssdk.go -typesOnly=false
```

Comment:
The SDK team started using the codegen tag in their generators, so we need this in order to run the generate commands. I put a `+build codegen` in `awssdk.go` as well.